### ISSUE TITLE
what would be url for swagger-ui.html incase of spring-java demo

### ISSUE THREAD

**Issue by sivamrkumar** [springfox/springfox-demos#29]
Hi Dilip,
I have followed your demo code for springmvc (based on 3.2.xx) version and used webjars dependency . I am able to get the json content for my controllers when I use /context-root/v2/api-docs.

But If I am using /context-root/swagger-ui.html , all I am getting is an empty page with a link to swagger homepage and 2 text boxes and an explore link.

Not sure what I am missing here? Could you please guide me on how to configure the swagger-ui similar to PetStore example of springBoot app incase of a springmvc application.

Thanks a ton in advance.

**Comment by dilipkrish** [comment]
Have you seen the [readme](https://github.com/springfox/springfox-demos/tree/master/spring-java-swagger)?

**Comment by sivamrkumar** [comment]
Yes, I did. I am able to open the swagger-ui.html but it is empty. I am able to get the .json content for my apis using /v2/api-docs/ . I wanted to get the nice UI display of this json content similar to swagger-ui petstore example. 

I don't want to deploy a separate webapp with swagger UI distribution and it's index.html pointing to my applications /v2/api-docs/ .

Just wanted to know how to fix the empty swagger-UI.html within my webcontext to show my api in nice UI format.

**Comment by sivamrkumar** [comment]
Hi Dilip,
Please show some light on this issue? Please look at my comment above for the explanation of the issue that I am facing. Thanks.

**Comment by dilipkrish** [comment]
Are you checking this address `http://localhost:8080/spring-java-swagger/swagger-ui.html`?

**Comment by lfallo1** [comment]
Were you able to solve this?  Same issue over here.  I am able to generate the documentation, and see it at v2/api-docs. The swagger-ui.html page, however, is blank aside from the swagger link, two text boxes, and an explore button.  The console does not indicate any errors.

**Comment by dilipkrish** [comment]
I think 2.5.0 might be the last supported release for spring 3.2.x

**Comment by lfallo1** [comment]
Thanks for the response, and it appears it was a versioning issue (however, I am using spring-boot 1.4.1 which looks like translates to spring 4.3.3 at the moment).

Regardless, it did begin working after downgrading swagger and swagger-ui each to 2.2.2

**Comment by ttww** [comment]
Downgrading swagger and swagger-ui to 2.2.2 worked also for me.  👍 
Plans for fixing?
2.2.2 don't know nothing about springfox.documentation.builders.ApiInfoBuilder().**contact**()...
Thanks for your tip.

**Comment by einverne** [comment]
I met the same problem, it seems latest Swagger2 is not support Spring 3.2.x. I have upgrade my project to Spring MVC 4.x. And everthing seems OK now.

**Comment by dilipkrish** [comment]
@einverne the last version supporting 3.2.x is version 2.5.0 I believe
